etsin_finder/frontend/js/stores/view/qvain/qvain.provenances.js

```diff
@@ -48,10 +48,22 @@ export const provenanceNameSchema = yup.object().shape({
   en: yup.string().typeError('qvain.validationMessages.history.provenance.nameRequired'),
 })
 
+const dateTest = yup.string().test('testStartDate', value => {
```

Comment:
I think that `d{1}` should be `\d`. Currently you're matching with a literal `d`, e.g. `dddd-03-0dT2d:3d:5dZ` will match.

Maybe extend `validateStringDate` in `extendYup.js` and rearrange it a bit? You could have an optional parameter that would allow the field to include time. It uses `date-fns/parse` which is more readable and less error-prone than regex.
```
function validateStringDate({allowTime}) {
   ...
   const parsed = parseDate(value, 'yyyy-MM-dd', new Date())
   let fail = !parsed || Number.isNaN(parsed.getTime())
   if (fail && allowTime) {
     ...try to parse date + time...
   ...etc...
```


Then you could use it like this
```
startDate: yup.string().date({allowTime: true}),
endDate: yup.string().date({allowTime: true}),
```